src/common/components/ui/formik/UploadFieldCertificates.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import { FormGroup, FileUpload } from '@patternfly/react-core';
+import { UploadFieldProps } from './types';
+import { getFieldId } from './utils';
+import HelperText from './HelperText';
+import { useTranslation } from '../../../hooks/use-translation-wrapper';
+import { useField } from 'formik';
+
+const UploadField: React.FC<UploadFieldProps> = ({
```

Comment:
Could you rename this component (and the whole file) to `CertificatesUploadField` or something similar? It would be more consistent with the naming pattern we have here.